www/src/components/PageEditBtn/constants.js

```diff
@@ -0,0 +1,35 @@
+const PATH_TO_EDIT_PAGE = 'https://github.com/openedx/paragon/edit/master/src';
+
+const componentsNamesMap = {
```

Comment:
This solution works but it will be hard to maintain in the future, because we will need to keep this object in sync with actual file structure in the repo. We will definitely forget about it at some point and have bugs eventually 😅.

A better solution would be to try to utilize gatsby's potential to its fullest 🙂. Gatsby builds these pages based on `.md` / `.mdx` files we have, so it knows their location at build time, we should use that to build URLs for the edit button, i.e. request file's path in [graphql](https://github.com/openedx/paragon/blob/master/www/utils/createPages.js#L13) query we have in `createPages` hook (by adding `fileAbsolutePath` in `node` object), generate URL for this specific file and pass it in `context` object through `createPage` function to React component that renders the component page. Then use it in the component.

This way we will not encounter any problems if we change file name's / structure in the future.

Also, I think we also need to add this functionality on non-component's pages (e.g. colors etc.), but perhaps in another PR.

Reply:
> Also, I think we also need to add this functionality on non-component's pages (e.g. colors etc.), but perhaps in another PR.

Ah, good catch. The "Leave feedback" CTA could also probably be added to non-component page headers, too (agree as a separate PR).

Reply:
Ok, thanks